applications/common/backend/src/main/resources/libs/kyanite/common/components/table/tablecell/editConfig/.content.json

```diff
@@ -0,0 +1,3 @@
+{
+  "reloadOnUpdate": "parent"
+}
```

Comment:
Thanks @paweljozwicki  for the tip. This works and is necessary in scenarios such as:
 - the Author modified "Columns" property of a cell
 - then, we need to reload other cells in the table row, so that they have a chance to adjusts their colspans so that the whole row can still have total colspan = 12